### DISCUSSION TITLE
Extester Launches VScode with persistence info from previous runs

### DISCUSSION THREAD

**Discussion by swathisuja** [redhat-developer/vscode-extension-tester#1467] [General]
We are trying to automate a VS code extension using Extester and
FIrst step is to install other extensions from default extensions. This data is persisting thought different runs of tests. 

We have cleared code cache, TEST_RESOURCES folder, yarn cache, .vscode extensions from user directory, despite all these, whenever we run tests, VS code is launched with other extensions pre-installed from previous run. 

Can you help us to remove the cache to have fresh test instance every time we run tests locally ?

**Comment by djelinek** [comment]
Hello, it should be enough to remove test resources folder in case you are not specifying custom path for extensions install.

Anyway, I would recommend try to define custom path for extensions-dir and then after tests to remove that folder, it could help

`-e, --extensions_dir <extensions_directory>  VS Code will use this directory for managing extensions`
